cmd/cosign/cli/options/attach.go

```diff
@@ -57,6 +58,8 @@ func (o *AttachSignatureOptions) AddFlags(cmd *cobra.Command) {
 			"signing certificate and end with the root certificate. Included in the OCI Signature")
 	cmd.Flags().StringVar(&o.TimeStampedSig, "tsr", "",
 		"path to the Time Stamped Signature Response from RFC3161 compliant TSA")
+	cmd.Flags().StringVar(&o.RekorBundle, "rekor", "",
```

Comment:
Can we name the flag `rekor-response`? This helps us to distinguish from the upcoming "sigstore bundle" format we plan to integrate.

Reply:
Hi, what is the sigstore bundle 👀

Reply:
https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto - sigstore-go and other newer clients use this rather than have separate outputs for cert, sig, rekor, etc